### PR TITLE
feat: Add channel & message URL formatters

### PR DESCRIPTION
**Pull request by Jiralite**
**Please describe the changes this PR makes and why it should be merged:**
Adds `channelLink()` and `messageLink()` which formats a link to a channel or message respectively.

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

File: packages/builders/src/messages/formatters.ts [hunk 1]
```diff
@@ -207,6 +207,63 @@ export function formatEmoji<C extends Snowflake>(emojiId: C, animated = false):
...
+ * @param messageId - The message's id
+ * @param guildId - The guild's id
+ */
+export function messageLink<C extends Snowflake, M extends Snowflake, G extends Snowflake>(
+	channelId: C,
+	messageId: M,
+	guildId: G,
+): `https://discord.com/channels/${G}/${C}/${M}`;
+
+export function messageLink<C extends Snowflake, M extends Snowflake, G extends Snowflake>(
```
Comment (imranbarbhuiya) [inline packages/builders/src/messages/formatters.ts:259]: U can use these formatters in https://github.com/discordjs/discord.js/blob/798466a696babd0857454c1dac921605cf5894b2/packages/discord.js/src/structures/BaseChannel.js#L57

**Review by imranbarbhuiya — CHANGES_REQUESTED** [review]

**Review by almeidx — COMMENTED** [review]
These should be re-exported in discord.js and used in the Message#url and Channel#url getters

**Comment by Jiralite** [comment]
> These should be re-exported in discord.js and used in the Message#url and Channel#url getters

This should be done with c19f6d9a9b046dc7c6e8f17983f07f4cfa6da8e3.

**Comment by Syjalo** [comment]
Maybe you will add also for users?
`https://discord.com/users/:id`

**Review by iCrawl — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**